scripts/domaintools.js

```diff
@@ -1,85 +1,38 @@
-function getUrlByParameter(name){
-    var url=window.location.href;
-    var indexOfEnd=url.length;
-    var index=url.indexOf(name);
-    var length=name.length;
-    return url.slice(index+length+1,indexOfEnd);
+function appendToParent (id, item, text_before, parent, show_item, text_after) {
+  if (item) {
+    if (show_item) {
+      parent.append($(id).text(text_before + item + text_after));
+    } else {
+      parent.append($(id).text(text_before + text_after));
+    }
+  }
+  return $(id);
```

Comment:
You don't need to return anything from `appendToParent`.
You don't use the outputs of this function anywhere.